Nitrox.Launcher/ViewModels/LaunchGameViewModel.cs

```diff
@@ -277,7 +283,7 @@ private bool ShouldSkipSteam(string args)
             return true;
         }
 
-        return false;
+        return true; // Default: skip Steam unless explicitly enabled
```

Comment:
Shouldn't it be the other way around? As it's only required when launching multiple instances which is a dev thing or for special use cases (like virtual split screen)